client/app/state/series/sagas/getSerie.js

```diff
@@ -0,0 +1,16 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { getSerieRequest, getSerieDone, getSerieError } from '../actions/getSerie';
+import { getSerieById } from '../../../services'
```

Comment:
aliases

Reply:
I have to investigate to add alias to expo to share this code between web and mobile